### PR TITLE
Tests: Add unit tests for Cairo Zero using Builtins

### PR DESCRIPTION
**Pull request by rodrigo-pino**

### CODE REVIEW

File: pkg/vm/builtins/bitwise.go [hunk 1]
```diff
@@ -26,14 +31,14 @@ func (b *Bitwise) InferValue(segment *memory.Segment, offset uint64) error {
 	xOffset := offset - bitwiseIndex
 	yOffset := xOffset + 1
 
-	xValue, err := segment.Read(xOffset)
-	if err != nil {
-		return err
+	xValue := segment.Peek(xOffset)
```
Comment (jmjac) [inline pkg/vm/builtins/bitwise.go:34]: Why the change from `Read` to `Peek`?

Comment (rodrigo-pino) [reply pkg/vm/builtins/bitwise.go:34]: To give a more helpful error message, if not the error message looks like:
"bitwise: bitwise: error_msg"

**Review by jmjac — APPROVED** [review]
Looks good
